CONTRIBUTING.rst

```diff
@@ -229,6 +230,7 @@ their individual contributions.
 * `marekventur <https://www.github.com/marekventur>`_
 * `Marius Gedminas <https://www.github.com/mgedmin>`_ (marius@gedmin.as)
 * `Markus Unterwaditzer <http://github.com/untitaker/>`_ (markus@unterwaditzer.net)
+* `Mathieu Paturel <http://github.com/math2001>`_ (mathieu.paturel@gmail.com)
```

Comment:
You'll need to rebase, because your other PR added your name here after this branch split off master.

Reply:
Hey. I've rebased from the current master branch (hypothesis' one). So, now, I need to push *force*, right? (I've tried a regular push, and it won't work, which makes sense).

I don't want to mess this up again...